sycl/include/sycl/ext/oneapi/experimental/cuda/masked_shuffles.hpp

```diff
@@ -0,0 +1,50 @@
+//==--------- masked_shuffles.hpp - cuda masked shuffle algorithms ---------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#if defined(__SYCL_DEVICE_ONLY__) && defined(__NVPTX__)
+
+namespace sycl {
+inline namespace _V1 {
+namespace detail {
```

Comment:
Since we only have a macro definition inside here, we don't need the namespaces.